gcc/rust/lex/rust-lex.cc

```diff
@@ -24,6 +24,9 @@
 #include "safe-ctype.h"
 
 namespace Rust {
+
+const char *kLexDumpFile = "gccrs.lex.dump";
```

Comment:
And would allow to move this to the session manager

Reply:
fixed